ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/projectimport/wizard/ImportProjectNotificationSubscriberImpl.java

```diff
@@ -51,12 +50,15 @@ public ImportProjectNotificationSubscriberImpl(CoreLocalizationConstant locale,
         this.notificationManager = notificationManager;
         this.workspaceId = appContext.getWorkspace().getId();
 
-        this.messageBus = messageBusProvider.getMessageBus();
+        messageBus = messageBusProvider.getMachineMessageBus();
```

Comment:
Check this line please, i think it not need, it because we init messgeBus only after starting ExtServer